src/Compilers/Core/Portable/SynthesizedLocalKind.cs

```diff
@@ -28,6 +29,12 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     internal enum SynthesizedLocalKind
     {
+        /// <summary>
```

Comment:
Can we just use `LoweringTemp` instead of adding a new kind? Usually, we don't need to distinguish between different kinds of short-lived temps introduced during lowering.

Reply:
No, we cannot. These can be ref structs and used in async methods.

Reply:
It's not clear from the description of LoweringTemp why would that be an issue.

Reply:
> No, we cannot. These can be ref structs and used in async methods. 

I do not see this enum member used in any meaningful way. We attach it to the local symbol, but never check for this value.